src/dom.js

```diff
@@ -270,3 +271,24 @@ export function childElementByTag(parent, tagName) {
     return el.tagName == tagName;
   });
 }
+
+
+/**
+ * Returns element attributes that pass the callback test as params.
+ * @param {!Element} element
+ * @param {function(!Element):string|null} parseAttributeCallback
+ * @return {Object}
+ */
+export function getAttributesAsParams(element, parseAttributeCallback) {
```

Comment:
I've had to write something close when I was implementing `data-params-` for iframes (which we decided not to do) and I think in brightcove there's a similar code that gets the `data-params-` attributes to pass them as params to the player URL. I wasn't sure this was the right place for it.